torch/csrc/jit/passes/onnx/eval_peephole.cpp

```diff
@@ -111,21 +112,21 @@ static void fuseConvBatchNorm(Block* b, ValueToParamPairMap& valsToParamsMap) {
         convNode->outputs()[i]->copyMetadata(bnNode->outputs()[i]);
       }
 
-      convNode->copyAttributes(*origconvNode);
+      convNode->copyAttributes(*origConvNode);
       convNode->insertBefore(bnNode);
-      convNode->addInput(origconvNode->inputs().at(0));
+      convNode->addInput(origConvNode->inputs().at(0));
 
-      auto conv_W = b->owningGraph()->addInput();
+      auto convW = b->owningGraph()->addInput();
```

Comment:
Suggestion: now that we are reformatting the names, can we also rename the params in this pass to have more helpful names?

Reply:
Tried to improve the names and add some comments that hopefully make the meaning clearer.